lib/context.js

```diff
@@ -75,10 +81,22 @@ class Context {
     const context = {
       addImport(source, imported) {
         return requires.add(source, imported);
+      },
+      addAbsImport(source, imported) {
```

Comment:
This turns out to be necessary for `addImport("whatwg-url")`.

Reply:
I’d&nbsp;prefer if&nbsp;`addImport(…)` supported&nbsp;package&nbsp;imports, which&nbsp;is&nbsp;what I’m&nbsp;doing in&nbsp;<https://github.com/jsdom/webidl2js/pull/162>.